spec/factories/user_factory.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :user do
+    username "felix_pip"
+    email "felix@pip.com"
+    status 1
+  end
+end
```

Comment:
I used:
factory :user do
    sequence(:name) { |n| "user name#{n}" }
    sequence(:email) { |n| "user email#{n}" }
  end
This was the way Elle had me set it out - she did suggest that I may want to have a check for the correct email format which I have yet to implement

Reply:
Everything looks good other than the two things Marta picked up - my comments are only for information - great work!